[docs] Hide account links and use a placeholder dash (#10251)
expo/expo: docs/pages/accounts/personal.md

### Personal Access Tokens

> To create Personal Access Tokens, you have to go to the [Expo Dashboard Settings access token page](https://expo.io/dashboard/[account]/settings/access-tokens). Anyone with this token can perform actions on behalf of your account. That applies to all content within your Personal Account.

Reviewer: ```suggestion
You can create Personal Access Tokens from the [Access Tokens page](https://expo.io/dashboard/[account]/settings/access-tokens) on your dashboard. Anyone with this token can perform actions on behalf of your account. That applies to all content within your personal account.
```

### Revoking Access Tokens

> In case a token was accidentally leaked, you can revoke them without changing your username and password. When you revoke the access token, you block all access to your account using this token. To do this, go to the [Expo Dashboard Settings access token page](https://expo.io/dashboard/[account]/settings/access-tokens), and delete the token you want to revoke.

Reviewer: ```suggestion
In case a token was accidentally leaked, you can revoke it without changing your username and password. When you revoke the access token, you block all access to your account using this token. To do this, go to the [Access Token page](https://expo.io/dashboard/[account]/settings/access-tokens) on your dashboard and delete the token you want to revoke.
```
